src/textual/app.py

```diff
@@ -858,14 +860,18 @@ def _display(self, renderable: RenderableType) -> None:
         """
         if self._running and not self._closed:
             console = self.console
-            if self._sync_available:
-                console.file.write("\x1bP=1s\x1b\\")
+            (
```

Comment:
This should probably be factored out in to two methods. Suggest `_begin_update` and `_end_update`